app/controllers/PasskeyController.scala

```diff
@@ -31,9 +31,14 @@ class PasskeyController(
     case Mode.Prod => "Janus-Prod"
   }
 
+  def showRegistrationPage: Action[AnyContent] = authAction {
+    implicit request =>
+      Ok(views.html.passkeyRegistration(request.user, janusData))
+  }
+
   private def apiResponse[A](
-      action: => Try[A]
-  )(implicit encoder: Encoder[A]): Result =
+                              action: => Try[A]
+                            )(implicit encoder: Encoder[A]): Result =
```

Comment:
Formatting has gone a bit awry here.
Running `sbt scalafmtAll` will sort it out.